docker/pulsar/Dockerfile

```diff
@@ -59,16 +59,16 @@ RUN apk add --no-cache \
        musl-dev \
        libffi-dev \
        py3-pip \
-       py3-grpcio \
        py3-yaml
 
 RUN pip3 install --break-system-packages \
         kazoo
 
 ARG PULSAR_CLIENT_PYTHON_VERSION
+# Once PyPI provides the grpcio-aarch64 package, we can remove https://wheel-index.linuxserver.io/alpine-3.19/
 RUN pip3 install --break-system-packages \
-    pulsar-client[all]==${PULSAR_CLIENT_PYTHON_VERSION}
-
+    pulsar-client[all]==${PULSAR_CLIENT_PYTHON_VERSION} \
+    --prefer-binary --find-links https://wheel-index.linuxserver.io/alpine-3.19/
```

Comment:
do we want to trust linuxserver.io for binaries?

Reply:
Sure, it provides the grpcio package.

Reply:
We also download this wheel file to local.

Reply:
From https://wheels.linuxserver.io/alpine-3.19/grpcio-1.62.2-cp311-cp311-linux_aarch64.whl to the pulsar repo.

Reply:
> From https://wheels.linuxserver.io/alpine-3.19/grpcio-1.62.2-cp311-cp311-linux_aarch64.whl to the pulsar repo.

how does that get referenced? does the Alpine image already reference that?

Reply:
> This solution comes from https://github.com/grpc/grpc/issues/27512#issuecomment-1030884361
> 
> Non Alpine official solution.

What makes this solution "official"?
I think a better solution is to lower the required grpc version in pulsar-client-python to 1.59.3 so that no external solution wouldn't be required at all.

The grpc version should match apache-bookkeeper-client's grpc version to ensure compatibility of generated stubs.